x-pack/plugins/cloud/public/index.ts

```diff
@@ -8,7 +8,9 @@
 import { PluginInitializerContext } from '../../../../src/core/public';
 import { CloudPlugin } from './plugin';
 
-export type { CloudSetup, CloudConfigType } from './plugin';
+export type { CloudSetup, CloudConfigType, CloudStart } from './plugin';
 export function plugin(initializerContext: PluginInitializerContext) {
   return new CloudPlugin(initializerContext);
 }
+
+export { Chat, LazyChat } from './components';
```

Comment:
Does `LazyChat` need to be exported? I don't think it is used anywhere directly

Reply:
Nope, will remove.